### PR TITLE
1363 consistent layout lib

### PR DESCRIPTION
**Pull request by ricardovdheijden**
#1363 

- Applies a consistent layout focussed on the header section.
- Using a settings file for tsup
- Enables Sourcemaps

### CODE REVIEW

File: packages/orchestrator-ui-components/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "scripts": {
         "test": "jest",
         "generate-version": "node -p \"'export const ORCHESTRATOR_UI_LIBRARY_VERSION = ' + JSON.stringify(require('./package.json').version) + ';'\" > src/configuration/version.ts && prettier --write src/configuration/version.ts",
-        "build": "npm run generate-version && tsup src/index.ts --format esm --dts --tsconfig tsconfig.build.json",
+        "build": "npm run generate-version && tsup src/index.ts",
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/package.json:27]: What does this do? Or why was it there?

File: packages/orchestrator-ui-components/src/components/WfoContentHeader/WfoContentHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    subtitle?: ReactElement | string;
+    children?: ReactElement | ReactElement[] | string;
+};
+
+export const WfoContentHeader: FC<WfoContentHeaderProps> = ({
+    title,
+    subtitle,
+    children,
+}) => {
+    return (
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoContentHeader/WfoContentHeader.tsx:23]:
```suggestion
}) => (
```

File: packages/orchestrator-ui-components/src/components/WfoContentHeader/WfoContentHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                    <WfoRenderElementOrString>
+                        {subtitle}
+                    </WfoRenderElementOrString>
+                </>
+            )}
+
+            <EuiSpacer size="l" />
+        </>
+    );
+};
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoContentHeader/WfoContentHeader.tsx:59]:
```suggestion
```

**Review by DutchBen — APPROVED** [review]

File: packages/orchestrator-ui-components/package.json (hunk 1 above)
Comment (ricardovdheijden) [reply packages/orchestrator-ui-components/package.json:27]: The removed parameters are moved to a separate file now `tsup.config.ts`

File: packages/orchestrator-ui-components/src/components/WfoContentHeader/WfoContentHeader.tsx (hunk 2 above)
Comment (ricardovdheijden) [reply packages/orchestrator-ui-components/src/components/WfoContentHeader/WfoContentHeader.tsx]: Done

**Review by wouter1975 — APPROVED** [review]

**Pull request merged**